src/Controller/LoginController.php

```diff
@@ -32,16 +32,13 @@ public function loginAction() {
 
         $user = $this->getUser($userName);
 
+

```

Comment:
Pabūsiu įkyrus, nes tokie pakeitimai blaškys tavo ateities kolegas: Why new line?

Reply:
Nutrinant nereikalingas eilutes (šis komentaras tinka ir kitiems atvejams, todėl tik čia užkomentuoju), netyčiom palikau dvi tuščias eilutes, o mano autoformatavimas NetBeans'e nelabai kreipia dėmesį ar yra tuščia eilutė  (tam tikrose vietose tik padaro), taip pat ignoruoja kiek tų tuščių eilučių yra, o Git Hub pagauna tokius pokyčius todėl tu ir pamatei. Spėčiau pas tave yra šiek tiek griežčiau suprogramuotas visas kodo formatavimas dėl to ir išvengi kitų klaidų ?

Reply:
Kodo formatavimo idealiai nesukonfigūruosi, nes kartais new line naudojami būten kažkokiems logiškai mažiau susijusiems blokams atskirti, o tokius atvejus IDE dar nemoka aptikti. Bet yra vienas labai paprastas būdas tokių pakeitimų sumažinti/išvengti: supushinus savo pakeitimus galima pačiam ateiti į github ir peržvelgti (kas šiaip jau yra "geroji praktika" ir rekomenduotina ne vien dėl tuščių eilučių)